Project.toml

```diff
@@ -2,7 +2,7 @@ name = "RoME"
 uuid = "91fb55c2-4c03-5a59-ba21-f4ea956187b8"
 keywords = ["SLAM", "state-estimation", "MM-iSAM", "MM-iSAMv2", "inference", "robotics"]
 desc = "Non-Gaussian simultaneous localization and mapping"
-version = "0.20.1"
+version = "0.21.0"
```

Comment:
```suggestion
version = "0.20.2"
```

Reply:
Hi @Affie , could we just do a patch release here or are there breaking changes?

Reply:
dont think there were any further changes.  Working to get Jim unstuck so going to rush merge this change too as a patch release.  I pretty confident there are no breaking changes that require a minor version release since v0.20.1 that was just release a few days ago.